File: timelogs.java
public class timelogs {
    public static double targetLogsPerDay = 7.5;
    public static double currentLogs = 4.6333;
    public static double remainingLogs = targetLogsPerDay - currentLogs;

    public static void main(String[] args) {
        System.out.println("Target Logs Per Day: " + timelogs.targetLogsPerDay);
        System.out.println("Current Logs: " + timelogs.currentLogs);
        System.out.println("Remaining Logs: " + timelogs.remainingLogs);

        timelogs();
    }




    public static void timelogs(){
        if(remainingLogs > 0){
            System.out.println("You have " + remainingLogs + " logs remaining.");
        } else {
            System.out.println("You have " + remainingLogs + " logs remaining.");
        }
    }
    

    public void createStatus(){
        System.out.println("Target Logs Per Day: " + timelogs.targetLogsPerDay);
        System.out.println("Current Logs: " + timelogs.currentLogs);
        System.out.println("Remaining Logs: " + timelogs.remainingLogs);
    }

}

File: timelogs.js
const targetLogsPerDay = 7.5;
const currentlogs = 3.6333;
const allocatedHours = 3;


const remainingLogs = targetLogsPerDay - currentlogs;
// console.log(`Remaining logs to reach the target: ${remainingLogs}`);

// for (let i = 0; i < 5; i++){
//     console.log(" ");
//     for (let j = i; j < 5; j++){
//         console.log("*");
//     }
// }


let rows = 5;

for (let i = 1; i <= rows; i++) {
    let rowStr = "";
    for (let j = 1; j <= i; j++) {
        rowStr += "*";
    }
    console.log(rowStr);
}